src/app.py

```diff
@@ -161,11 +162,17 @@ def create_app(environment="Development"):
     )
 
     api.add_resource(LoggedActivityApprovalAPI,
-        "/api/v1/approve/logged-activities",
-        "/api/v1/approve/logged-activities/",
+        "/api/v1/logged-activities/approve",
```

Comment:
Have you communicated this change to Chris?

Reply:
Yes.